README.md

```diff
@@ -115,11 +113,9 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
-[scala-client-img]: https://github.com/snowplow/iglu/wiki/technical-documentation/images/iglu-clients.png
-
-[iglu-wiki]: https://github.com/snowplow/iglu/wiki
+[iglu-docs]: https://docs.snowplow.io/docs/pipeline-components-and-applications/iglu/
 [snowplow-schema-post]: http://snowplowanalytics.com/blog/2014/06/06/making-snowplow-schemas-flexible-a-technical-approach/
-[resolver-config]: https://github.com/snowplow/iglu/wiki/Iglu-client-configuration
+[resolver-config]: https://docs.snowplow.io/docs/pipeline-components-and-applications/iglu/iglu-resolver/
```

Comment:
nit - since some of the links are updated, could we also switch the rest of them to `snowplow.io` domain as well?